app/views/devise/passwords/edit.html.slim

```diff
@@ -0,0 +1,17 @@
+- content_for(:description) do
+  h2 = t('devise.passwords.description')
+
+= form_with(model: resource, url: password_path(resource_name), local: true, html: { method: :put }) do |f|
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/nimblehq/nimble-survey-web/blob/801dc79f134840871ce63769348d824626712f34/app/views/devise/passwords/edit.html.slim#L4"></span>Line is too long. [107/80]</td>
    </tr>
  </tbody>
</table>